build.gradle.kts

```diff
@@ -138,19 +142,18 @@ testing {
         val integrationTest by registering(JvmTestSuite::class) {
             dependencies {
                 implementation(project())
-                implementation("org.springframework.boot:spring-boot-starter-test") {
+                implementation(libs.org.springframework.boot.spring.boot.starter.test) {
                     exclude("org.mockito", "mockito-core")
                     because("Use MockK instead of Mockito since it is better suited for Kotlin")
                 }
-                implementation("org.springframework.boot:spring-boot-starter-webflux")
-                implementation("com.ninja-squad:springmockk:4.0.0")
-                implementation("org.springframework.security:spring-security-test")
```

Comment:
Why could we remove this?

Reply:
spring-security-test is used in unit tests for `WithMockUser`, but not in integration tests.